samples/javascript_nodejs/51.teams-messaging-extensions-action/pages/CustomForm.html

```diff
@@ -0,0 +1,50 @@
+<html>
+<head>
+    <style>
+        .login {
+            text-align: center;
+        }
+    </style>
+    <title>Microsoft Teams ME Action WebView</title>
+</head>
+<body>
+    <script src="https://statics.teams.microsoft.com/sdk/v1.7.0/js/MicrosoftTeams.min.js"></script>
```

Comment:
Is this the suggested way to include the teams library? I would have guessed this `script` tag should go inside the `<head>` node. Probably should include `type="text/javascript"` as well.